### PR TITLE
Add the plugin for microsoft mssql-jdbc

### LINKED ISSUE #2340: When Skywallking supports MSSQL ?
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [1 ] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?
Currently our company use MSSQL , so I want to know when the MSSQL plugin will be supported ? thanks
___
### Bug
- Which version of SkyWalking, OS and JRE?
6.0.0GA , JRE1.8.062
- Which company or project?

- What happen?
If possible, provide a way for reproducing the error. e.g. demo application, component version.

___
### Requirement or improvement
- Please describe about your requirements or improvement suggestions.

### PR DESCRIPTION
**Pull request by zifeihan**
### Add an agent plugin to support mssql-jdbc
- [x] Add a test case for the new plugin, refer to [the doc](https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md)
- [x] Add a component id in [the component-libraries.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml)
- [x] Add a logo in [the UI repo](https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets)

Fix #2340, add an agent plugin to support mssql-jdbc,
image
image
image

### CODE REVIEW

File: test/plugin/scenarios/mssql-jdbc-scenario/support-version.list [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+6.2.2.jre8
+6.4.0.jre8
+6.5.2.jre8-preview
+7.0.0.jre8
+7.2.2.jre8
+7.4.1.jre8
+8.2.2.jre8
+8.4.0.jre8
+9.1.0.jre8-preview
```
Comment (kezhenxu94) [inline test/plugin/scenarios/mssql-jdbc-scenario/support-version.list:25]: I prefer not to test `preview` version

Comment (zifeihan) [reply test/plugin/scenarios/mssql-jdbc-scenario/support-version.list]: fixed.

File: CHANGES.md [hunk 2]
```diff
@@ -18,7 +18,8 @@ Release Notes.
 * Add the plugin for async-http-client 2.x
 * Fix NPE in the nutz plugin.
 * Provide Apache Commons DBCP 2.x plugin.
-* Add the plugin for mssql-jtds 1.x plugin.
+* Add the plugin for mssql-jtds 1.x.
+* Add the plugin for mssql-jdbc.
```
Comment (wu-sheng) [inline CHANGES.md:22]: Version missed here.

Comment (zifeihan) [reply CHANGES.md]: done.

**Review by kezhenxu94 — APPROVED** [review]
LGTM, thanks

**Review by wu-sheng — APPROVED** [review]

**Review by dmsolr — APPROVED** [review]
LGTM

**Pull request merged**